src/workers/airtable-sync/actions/updateMessageContent.js

```diff
@@ -63,6 +63,33 @@ module.exports = async function updateMessageContent(record) {
       requestFields.status
     )} => ${statusBadge}`
   );
+
+  const streetsLineHeading = `*${str(
```

Comment:
now that there's a bit more logic in updateMessageContent, can we split this out into it's own function so its a bit more readable?

Reply:
Just updated, lemme know if that's what you meant